### ISSUE TITLE
[Bug] Browse/Latest goes to top of screen when backing out of manga

### ISSUE THREAD

**Issue by DKTsubasa** [Jays2Kings/tachiyomiJ2K#110]
### Device information
* Tachiyomi version: 0.9.8 (or the newest as of February 2020)
* Android phone & version: LG Phoenix 4, Android 8.1.0

## Steps to reproduce
1. Click Catalogues
2. Click on an extension, or Latest in any extension (I mostly noticed this on MangaDex, tested on others)
3. Scroll down and select a manga, then back out of manga (make sure to remember what was at the top before scrolling down)

### Expected behavior
Screen should stay in same place as before clicking on a manga

### Actual behavior
Screen jumps back to the top of the list when you back out of any manga, even if you scrolled a long way down

### Other details
So basically, after the new update, I noticed that everytime I look at manga in the catalogues, if I click on one thats let's say 5 from the top, everything is fine. But, if i back out of that manga to look at more, it puts me back at the top of the page. An example would be you clicking on something at the bottom of a webpage, then going back and the webpage loads at the top without remembering where you were

**Comment by Jays2Kings** [comment]
Fixed as of 0.9.81
